.github/SUPPORT.md

```diff
@@ -0,0 +1,10 @@
+
+# Looking for support? We want to help
+
+Please post your question to X (#) using the Timber tag.
+
+## Other places
+
+You can use [GitHub Discussions](#) to open up questions and engage in discussions with the contributors and other users.
+
+Please don't post to the [EXAMPLE forum](#). You might see a response, but it will just redirect you to either GitHub (for bugs/issues) or StackOverflow (for usage/support questions).
```

Comment:
Do we need this line? Don't post somewhere? Stackoverflow? 🤔

Reply:
I removed the SUPPORT.md file because the file seemed like a template with no real information, and we already cover the various channels through which people can contact us in the README, such as Google Groups, Zulip, etc.